docker/setup.cfg

```diff
@@ -0,0 +1,35 @@
+[metadata]
+name = deeplabcut-docker
+version = attr: deeplabcut_docker.__version__
+author = A. & M. Mathis Labs
+author_email=alexander@deeplabcut.org
+description = A helper package to launch DeepLabCut docker images
+url = https://github.com/DeepLabCut/DeepLabCut/docker
+project_urls =
+    Bug Tracker = https://github.com/DeepLabCut/DeepLabCut/issues 
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)
+    Operating System :: MacOS
+    Operating System :: POSIX :: Linux
+    Topic :: Utilities
+license = LGPLv3
+
+[options]
+package_dir =
+    = . 
+py_modules = deeplabcut_docker
+python_requires = >=3.6
```

Comment:
3.6 is outdated in DLC, should read at lease 3.7

Reply:
The rationale here would be that Python 3.6 is still the standard distribution on Ubuntu/Debian systems; Inside the docker containers, Python 3.8 is used to meet the DLC requirement---this here is only for the helper script allowing to pull and launch containers.

If we increase the version here, users with older ubuntu systems would not be able to pull the images via `deeplabcut-docker`.

Should we keep 3.6 here, or anyways bump to 3.7?

Reply:
(it might also be worth checking what macOSX users typically have on their systems)

Reply:
we release conda file with forced 3.7 for all OS; fine to leave for 3.6, thanks for explanation!